src/test/resources/org/jenkinsci/plugins/tuleap_git_branch_source/jcasc/jenkins.yml

```diff
@@ -0,0 +1,12 @@
+credentials:
+  system:
+    domainCredentials:
+      - credentials:
+          - tuleapAccessToken:
+              scope: SYSTEM
+              id: "whatever"
+              token: "tlp-k1-24.2dd47d2e9280a430f4eb862cfbc85e90e8b286818dcbe85bd60d8feb7162580a"
+
+unclassified:
+  tuleapConfiguration:
+    domainUrl: https://my.tuleap.instance.net
```

Comment:
Is it possible to use the example.net domain instead ?

Reply:
sure, as long as there isn't any overlap with DEFAULT_TULEAP_DOMAIN_URL anything is possible :)